cms/utils/page.py

```diff
@@ -150,7 +150,7 @@ def get_page_from_request(request, use_path=None, clean_path=None):
 
     draft = use_draft(request)
     preview = 'preview' in request.GET
-    path = request.path_info if use_path is None else use_path
+    path = request.path if use_path is None else use_path
```

Comment:
we currently use `path_info` everywhere, so I guess more clarifications why this is needed here would be needed

Reply:
When using djangocms with a defined script_name and using an apphook this one does not return the correct url because of Path_info, the prefix is ​​missing.